### PR TITLE
Doc update for ocp 4.14

### PR DESCRIPTION
**Pull request by ShellyXueHan**
- remove deploymentConfig reference
- update dc intro to deployment
- add "Logging in to the CLI using a web browser"
- misc fixes
- note for Steven to update details about the pruner specifications

### CODE REVIEW

**Review by StevenBarre — CHANGES_REQUESTED** [review]
DeploymentConfigs are deprecated, not removed yet. So the automation and pruning will continue to target DCs as long as they are supported by the Kube API.

File: src/docs/build-deploy-and-maintain-apps/deploy-an-application.md [hunk 1]
```diff
@@ -117,14 +117,14 @@ Infrastructure as Code (IaC) is a best practice for managing and provisioning te
...
+Deployment in OpenShift are a recipe for deploying your application. They consist of the following are considered as most important field:
 
+- **Strategy:** Defines the strategy used for updating the Deployment. The default strategy is RollingUpdate, where new pods are gradually rolled out while old pods are terminated. Recreate is the other option, where it deletes all old Pods before creating new ones
 - **Pod Template:** This defines the desired state of the pods being deployed, including the base image, ports to expose, and environment variables
 - **Replicas:** The number of instances of your application that should be maintained
 - **Selectors:** Labels that identify the pods managed by the deployment configuration
-- **Triggers:** Events that will cause a new deployment. `DeploymentConfig` and `BuildConfig` both support image triggers and configuration change triggers. Additionally, Deployments and StatefulSets can also use [image triggers](https://docs.openshift.com/container-platform/4.12/openshift_images/triggering-updates-on-imagestream-changes.html) which initiate updates when an image stream tag that the triggering resource points to is updated.
```
Comment (StevenBarre) [inline src/docs/build-deploy-and-maintain-apps/deploy-an-application.md:127]: I think we want to keep part of this?
> Additionally, Deployments and StatefulSets can also use [image triggers]

still seems relevant

File: src/docs/platform-automation/pruning.md [hunk 2]
```diff
@@ -24,17 +24,18 @@ Last updated: **March 8, 2024**
...
 * **[Related pages](#related-pages)**
 
-## DeploymentConfigs
+## Deployments
 
-When a `DeploymentConfig` rolls out a new version, it creates a new `ReplicationController`. Over time, the accumulation of these objects can happen. Therefore, older ones are routinely removed on a daily basis through pruning as well.
+When a `Deployment` rolls out a new version, it creates a new `ReplicaSet`. Over time, these ReplicaSets can accumulate as each new deployment creates a new one. OpenShift automatically manages the history of ReplicaSets by retaining a configurable number of old ReplicaSets and scaling them down to 0 replicas. By default, older ReplicaSets are not removed but are instead kept in a scaled-down state. However, you can configure a retention policy to limit the number of old ReplicaSets retained, or manually prune them if needed.
 
 A few settings are passed to the pruner job:
+<!-- Steven please update this part: -->
```
Comment (StevenBarre) [inline src/docs/platform-automation/pruning.md:38]: https://docs.openshift.com/container-platform/4.14/applications/pruning-objects.html#pruning-deployments_pruning-objects

> To also prune replica sets associated with Deployment objects, use the --replica-sets flag. **This flag is currently a Technology Preview feature.**

Since its tech preview we haven't turned that on yet. So only DCs are getting pruned, not Deploys

Comment (ShellyXueHan) [reply src/docs/platform-automation/pruning.md]: kk reverted the changes on this part!

File: src/docs/platform-automation/automated-scaling.md [hunk 3]
```diff
@@ -27,24 +27,24 @@ To tackle this, we've created automated tools to keep an eye on apps on the plat
 
 Our tools specifically check for:
 
-- `Deployments` and `deploymentconfigs` relying on images that haven't been updated in more than a year
-- Constantly crashing `Deployments`, `deploymentconfigs`, and `statefulsets`
+- `Deployments` relying on images that haven't been updated in more than a year
+- Constantly crashing `Deployments` and `statefulsets`
```
Comment (StevenBarre) [inline src/docs/platform-automation/automated-scaling.md:31]: Can we revert these changes too?

Comment (ShellyXueHan) [reply src/docs/platform-automation/automated-scaling.md]: ah right missed that one!

**Review by StevenBarre — APPROVED** [review]
